Tensile/KernelWriterAssembly.py

```diff
@@ -3186,13 +3187,17 @@ def functionSignature(self, kernel ):
           msg = "reading and writing LDS at same time require 2 LDS buffer"
       elif self.overflowedResources == 6:
         msg = "SIA2 better with occupancy 2"
+      elif self.overflowedResources == 7:
+        msg = "too many accvgprs"
       else:
         msg = "unknown"
 
       if globalParameters["PrintSolutionRejectionReason"]:
-        printWarning("%s overflowed resources.  errorCode=%d, msg=\"%s\", vgprs=%u, sgprs=%u" \
-          % (self.kernelName, self.overflowedResources, msg, \
-          self.vgprPool.size(), self.sgprPool.size()))
+        printWarning(
+            f"{self.kernelName} overflowed resources. errorCode={self.overflowedResources}" \
+            f", msg='{msg}', vgprs={self.vgprPool.size()}, sgprs={self.sgprPool.size()}" 
```

Comment:
Don't we need + here?
Could you double-check if the message format here is same as before change (except for acc part)?

Reply:
The plus allows a simple inline if/else to be used to control whether accvgprs=... is printed. I will verify for good measure.

Reply:
@nakajee confirmed the following output with the latest commit,
```bash
# after
UserWarning: Cijk_Alik_Bljk_SB_MT16x16x64_MI16x16x4x1_... overflowed resources. errorCode=0, msg='too many accvgprs', vgprs=256, sgprs=66, accvgprs=0
# before
UserWarning: Cijk_Alik_Bljk_SB_MT16x16x64_MI16x16x4x1_... overflowed resources.  errorCode=0, msg="too many accvgprs", vgprs=256, sgprs=66
```

Reply:
Thanks for confirmation.
I thought we needed "+" to concatenate 2 strings (I am not very good at Python though)...
I am OK as long as you confirmed it worked as expected.